analyzer/testdata/src/constructor_check/constructors_after_struct_methods.go

```diff
@@ -0,0 +1,24 @@
+package simple
+
+//nolint:recvcheck // testing linter
```

Comment:
I propose to disable linting for the whole `testdata` directory in `.golangci.yml`. See `exclusions` in https://golangci-lint.run/usage/configuration/#linters-configuration

Reply:
this is useless: `testdata` is already excluded.

Reply:
this is useless: `testdata` is already excluded.